docs/user-guide/src/install.md

```diff
@@ -2,16 +2,20 @@
 
 ## Prusti Assistant
 
-The easiest way to try out Prusti is by using the ["Prusti Assistant"](https://marketplace.visualstudio.com/items?itemName=viper-admin.prusti-assistant) extension for [Visual Studio Code](https://code.visualstudio.com/). Please confer the extension's webpage for detailed installation instructions.
+The easiest way to try out Prusti is by using the ["Prusti Assistant"](https://marketplace.visualstudio.com/items?itemName=viper-admin.prusti-assistant) extension for [Visual Studio Code](https://code.visualstudio.com/). Please confer the extension's webpage for:
+* Detailed installation and first usage instructions.
+* The description of the available commands, among which commands to run and update the verifier.
+* The description of the configuration flags.
+* Troubleshooting instructions.
 
-To update the currently installed version of Prusti within Prusti Assistant, it suffices to open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and run the command `Prusti: update dependencies`.
-
-> **Warning:** Some of the Prusti-specific syntax described in this guide is currently only available in the "nightly" build channel, which corresponds to the latest development version of Prusti.
+> **Warning:** Some of the Prusti-specific syntax described in this guide is currently only available in the "LatestDev" build channel, which corresponds to the latest development version of Prusti.
 > The settings for switching to this version in Prusti Assistant are found in
 > `File → Preferences → Settings → Prusti Assistant → Build Channel`.
 
+In case you experience difficulties or encounter bugs while using Prusti assistant, please [open an issue in Prusti Assistant's repository](https://github.com/viperproject/prusti-assistant/issues) or contact us in the [Zulip chat](https://prusti.zulipchat.com/).
```

Comment:
```suggestion
In case you experience difficulties or encounter bugs while using Prusti Assistant, please [open an issue in Prusti Assistant's repository](https://github.com/viperproject/prusti-assistant/issues) or contact us in the [Zulip chat](https://prusti.zulipchat.com/).
```